src/theme/index.css

```diff
@@ -1,12 +1,14 @@
 :root {
   --primary-color: #1a1a1a;
   --secondary-color: #111;
-  --text-color: #fff;
+  --text-color: white;
   --none: none;
 }
 
 body {
   background: var(--primary-color) !important;
+  background-color: #1a1a1a;
+  background: #1a1a1a;
```

Comment:
Please remove these two as we already support the `background: var(--primary-color) !important;` on line 9.